acceptance/data/index.yaml

```diff
@@ -1,3 +1,17 @@
+# Copyright 2014 Google LLC
```

Comment:
Just to clarify, does the copyright always points to the year of the file's original creation? Even when it is added retroactively?

Reply:
Ideally, you should add this when you create the file.